lammps_ona/Untitled.ipynb

```diff
@@ -0,0 +1,101 @@
+{
```

Comment:
I would rename the notebook to something other than `Untitled`, like `lammps_ona.ipynb`.